frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -294,15 +294,18 @@ function ReaderCoptListener:getAltStatusBarMenu()
             },
             {
                 text_func = function()
-                    local status = _("off")
+                    local status = _(" ")
                     if self.battery == 1 then
                         if self.battery_percent == 1 then
-                            status = _("percentage")
+                            status = _("(percentage)")
                         else
-                            status = _("icon")
+                            status = _("(icon)")
                         end
                     end
-                    return T(_("Battery status: %1"), status)
+                    return T(_("Battery status %1"), status)
```

Comment:
We usually separate the current value with a colon.

Reply:
I meant that translators will see ": icons", which is just not plain nice of asking of them.
And you using a `status = _(" ")` (which wouldn't need to be translated :)) will push the ▶ further from the text, and it will look different from others (more spacing). So all the constructions suck, and if there's not solution, just use 3 full strings:
```
_("Battery status")
_("Battery status: icon")
_("Battery status: percentage")
```